include/aspect/simulator_access.h

```diff
@@ -32,20 +32,6 @@
 #include <aspect/global.h>
 #include <aspect/parameters.h>
 #include <aspect/introspection.h>
-#include <aspect/material_model/interface.h>
-#include <aspect/geometry_model/initial_topography_model/interface.h>
-#include <aspect/geometry_model/interface.h>
-#include <aspect/gravity_model/interface.h>
-#include <aspect/boundary_temperature/interface.h>
-#include <aspect/boundary_composition/interface.h>
-#include <aspect/initial_temperature/interface.h>
-#include <aspect/initial_composition/interface.h>
-#include <aspect/boundary_velocity/interface.h>
-#include <aspect/boundary_traction/interface.h>
-#include <aspect/mesh_refinement/interface.h>
-#include <aspect/postprocess/interface.h>
-#include <aspect/heating_model/interface.h>
-#include <aspect/adiabatic_conditions/interface.h>
```

Comment:
Probably the latter. But since `simulator_access.h` compiles, I suspect that it is ok to not move the includes there. 

`simulator_access.cc` in many cases doesn't actually need to know the *complete* type because it just forwards references. It is ok for these functions to know that the type exists, but a typical getter function that just returns a reference doesn't need to know the complete type of the thing it returns.